dev-test/githunt/types.preResolveTypes.onlyOperationTypes.ts

```diff
@@ -12,6 +12,50 @@ export type Scalars = {
   Float: number;
 };
 
+/** A comment about an entry, submitted by a user */
```

Comment:
@Borduhh this output changing looks like a regression on the `onlyOperationTypes` option 👀

Reply:
That would be. Found the typo and fixed it 🎉. There should be not regression now.